core/src/main/resources/lib/form/repeatable/repeatable.js

```diff
@@ -107,11 +107,11 @@ var repeatableSupport = {
         if (addButtonElements.length == 1 && this.enableTopButton) {
           buttonElement = addButtonElements[0];
           parentOfButton = buttonElement.parentNode;
-          var addTopButton = document.createElement("input");
+          var addTopButton = document.createElement("button");
           addTopButton.type = "button";
-          addTopButton.value =
-            buttonElement.textContent || buttonElement.innerText;
-          addTopButton.className = "repeatable-add repeatable-add-top";
+          addTopButton.innerHTML = buttonElement.innerHTML;
```

Comment:
Is there a potential this could be used for an XSS attack?

Reply:
I've tried to inject various things via `add` attribute and it didn't work out. Content was properly escaped, presumably because of `escape-by-default='true'`.